src/styles/blocks/modifiers.scss

```diff
@@ -0,0 +1,17 @@
+.stars--4 .stars__item:nth-child(-n + 4) {
+  background-image: url(../images/star-active.svg);
+}
+
+.card__button--styles {
+  color: white;
+  font-size: 14px;
+  font-weight: 700;
+  text-align: center;
+  line-height: 40px;
```

Comment:
The properties in CSS should be ordered alphabetically. It's a good practice to follow, as it makes your code more readable and easier to understand. [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules)